src/common/utils.py

```diff
@@ -327,6 +330,14 @@ def get_instance_properties(region, proxy_config, instance_type):
                     )
                 )
                 slots = vcpus
+
+            if slots > vcpus:
```

Comment:
there might be some remote cases when you want a node to appear bigger than it is.  we might want to avoid that check since using an integer in cfn_scheduler_slots is already a strong customization

Reply:
Removed